src/main/resources/db/migration/V31__behandlingsresultat.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE behandling ADD COLUMN resultat VARCHAR;
+UPDATE behandling SET resultat='IKKE_SATT';
```

Comment:
Hvis den ikke skal være nullable kan man sette
`ALTER TABLE behandling ADD COLUMN resultat VARCHAR NOT NULL DEFAULT 'IKKE_SATT';`

Reply:
Enig!